src/msg/async/ProtocolV1.cc

```diff
@@ -15,8 +15,10 @@
 #define dout_prefix _conn_prefix(_dout)
 ostream &ProtocolV1::_conn_prefix(std::ostream *_dout) {
   return *_dout << "--1- " << messenger->get_myaddrs().legacy_addr() << " >> "
-                << connection->peer_addrs->legacy_addr() << " conn("
-                << connection << (connection->msgr2 ? " msgr2" : " legacy")
+                << connection->peer_addrs->legacy_addr() << " " << this
+		<< " conn("
+                << connection << (connection->msgr2 ? " msgr2=" : " legacy=")
```

Comment:
can we delete this part (`msgr2=` or `legacy=`) since we already have the `--1-` or `--2-` in the beginning of the prefix.